src/NonVariantWrappers.dfy

```diff
@@ -0,0 +1,109 @@
+// RUN: %dafny /compile:0 "%s"
+
+// Assert that this file is minimally different from Wrappers.dfy,
+// to ensure they are mostly drop-in replacements for each other.
+// RUN: %diff "%S/Wrappers.dfy" "%s" | tail -n +3 > "%t" || true
+// RUN: %diff "%t" "%s".diff.expect
```

Comment:
Could we generate this file automatically?  Like we do for Options.txt in the Dafny repo.

Reply:
I'm sure we could, but I don't think it's worth it in this case. I don't expect these files to be changed often, and this is a relatively temporary solution until https://github.com/dafny-lang/dafny/issues/2013 is fixed anyway.

Reply:
I'm more convinced by the second argument than the first (I have heaps of stuff to add to `Wrappers.dfy` from `Dafny-in-Dafny`).

But, given the second argument, should we really have this file in the standard library (and not in whichever client code needs it temporarily?)